spec/features/creating_a_tag_plate_spec.rb

```diff
@@ -177,7 +187,7 @@
       let(:template_factory) { :v2_dual_index_tag_layout_template }
 
       context 'when nothing has been done' do
-        #let(:submission_pools) { json(:dual_submission_pool_collection) }
+        let(:enforce_uniqueness) { false }
```

Comment:
I'm not sure why I needed to add this.  Is it the correct expectation @andrewsparkes or are the mocks letting us down and so this should have been submitted as `true`?